### DISCUSSION TITLE
How can I see the correct analytics after adding a custom domain name?

### DISCUSSION THREAD

**Discussion by MyLittleRunaway** [vercel/community#2452] [Help] [answered]
### Summary

Hi. I'm wondering how I can see my analytics. 
I don't use the domain seen below to promote my site - I do it through another domain that I've linked (gptru.io)
I can't see the analytics for gptru.io - all I can see is the data for ruski-gpt.vercel.app 

<img width="1200" alt="Screenshot 2023-05-15 at 03 35 58" src="https://github.com/vercel/community/assets/78008842/524a4caa-c8bc-47d3-925c-4f8223faf75c">
<img width="1280" alt="Screenshot 2023-05-15 at 03 36 15" src="https://github.com/vercel/community/assets/78008842/6b2e4665-d519-484b-9832-bb04afc2a1b4">

**Comment by amyegan** [comment, marked as answer]
Hi @DrGunbot! Analytics for custom production domains should show up automatically after following the setup steps. Sometimes people select a custom branch which directs the domain to the Preview environment. Please verify that your custom domain is connected to the Production environment.
